app/scripts/controllers/account.box.dataupload.js

```diff
@@ -57,9 +57,8 @@
     function submitData () {
       vm.success = false;
       vm.error = '';
-
       OpenSenseMapAPI
-        .postMeasurements($state.params.id, vm.measurementData, vm.dataFormat)
+        .postMeasurements($state.params.id, vm.measurementData, vm.dataFormat,$state.params.box.access_token)
```

Comment:
lets keep it readable :)

```suggestion
        .postMeasurements($state.params.id, vm.measurementData, vm.dataFormat, $state.params.box.access_token)
```